tests/test_integration_8_days_snapshot_report/test_integration_8_days_snapshot_graph.py

```diff
@@ -0,0 +1,525 @@
+                            registration_event_datetime=registration_time.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                            event_type=EventType.TRANSFER_COMPATIBILITY_STATUSES.value,
+                            sendingSupplierName="EMIS",
+                            requestingSupplierName="TPP",
+                            payload=create_transfer_compatibility_payload(
+                                internalTransfer=False,
+                                transferCompatible=True,
+                                reason="test1"
+                            )
+                        )
+                    ),
+                    sourcetype="myevent",
+                )
+
+                # Awaiting Integration
+                index.submit(
+                    json.dumps(
+                        create_sample_event(
+                            conversation_id=f'ready_to_integrate_{idx}',
+                            registration_event_datetime=registration_time.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                            event_type=EventType.EHR_RESPONSES.value,
+                            sendingSupplierName="EMIS",
+                            requestingSupplierName="TPP",
+                        )
+                    ),
+                    sourcetype="myevent",
+                )
+
+                index.submit(
+                    json.dumps(
+                        create_sample_event(
+                            conversation_id=f'ready_to_integrate_{idx}',
+                            registration_event_datetime=registration_time.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                            event_type=EventType.READY_TO_INTEGRATE_STATUSES.value,
+                            sendingSupplierName="EMIS",
+                            requestingSupplierName="TPP",
+                        )
+                    ),
+                    sourcetype="myevent",
+                )
+
+            # Act
+            test_query = self.generate_splunk_query_from_report(
+                "gp2gp_integration_8_days_snapshot_report"
+                "/gp2gp_integration_8_days_snapshot_report_awaiting_integration_percentage"
+            )
+
+            test_query = set_variables_on_query(
+                test_query,
+                {
+                    "$index$": index_name,
+                    "$start_time$": report_start.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                    "$end_time$": report_end.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                    "$cutoff$": cutoff,
+                },
+            )
+
+            sleep(2)
+
+            telemetry = get_telemetry_from_splunk(
+                self.savedsearch(test_query), self.splunk_service
+            )
+            self.LOG.info(f"telemetry: {telemetry}")
+
+            # Assert
+            expected_values = {
+                "In flight": "50.00",
+                "Not integrated after 8 days": "50.00",
```

Comment:
For testing purposes I wouldn't do the same amount for each scenario as this could cause someone to miss a mistake if the percentages were the wrong way round. So maybe do 2 late ones and one on-time one.